services/QuillLMS/engines/evidence/app/views/evidence/research/gen_ai/datasets/show.html.erb

```diff
@@ -0,0 +1,55 @@
+<div>
+  <h2><%= @dataset.model_name.human.titleize %></h2>
+  <p><%= @dataset %></p>
+
+  <h3>Optimal Count</h3>
+  <p><%= @dataset.optimal_count %></p>
+
+  <h3>Suboptimal Count</h3>
+  <p><%= @dataset.suboptimal_count %></p>
+  <hr/>
+
+  <header>
+    <h2>
+      Trials
+      <%= link_to 'new', new_research_gen_ai_dataset_trial_path(@dataset), class: 'new-link', style: 'margin-right: 20px;' %>
+    </h2>
+  </header>
+
+  <br>
+
+  <% if @trials.any? %>
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Date</th>
+          <th>LLM</th>
+          <th>Activity</th>
+          <th>Conjunction</th>
+          <th>LLM Prompt Description</th>
+          <th># of Test Results</th>
+          <th>Status</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @trials.each do |trial| %>
+          <tr>
+            <td><%= trial.id %></td>
+            <td><%= trial.created_at.strftime("%m/%d/%y, %H:%M") %></td>
+            <td><%= trial.llm %></td>
+            <td><%= trial.stem_vault.name %></td>
```

Comment:
nil check?

Reply:
trial belongs_to :dataset, and dataset belongs_to :stem_vault so these should not require safe navigation.